app/mailers/user_notifier.rb

```diff
@@ -0,0 +1,13 @@
+class UserNotifier < ApplicationMailer
+	  default from: 'michael.foster@outlook.com',
+	  	return_path: 'michael.foster@outlook.com'
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.